src/include/firebird/Message.h

```diff
@@ -25,7 +25,15 @@
 
 #include "ibase.h"
 #include "./Interface.h"
+#if defined USE_SYSTEM_BOOST
```

Comment:
This is public header file. So I think name must be better `FB_USE_SYSTEM_BOOST`.